### PR TITLE
stronger/fix tests & python3.9 compatibility

### PR DESCRIPTION
**Pull request by theomonnom**
- also print serverside errors of 11labs
- add word tokenizer to 11labs (default to the basic one)
- stt maximum 0.2 WER
- try_trigger_generation in 11labs set to false (bad practice)
     - added chunk_length_schedule instead
     - allow multiple concurrent ws connections when multiple segments are pushed
- check stt events order
- google stt, fix unordered events & accuracy (now competitive with other stt)
- fix stt StreamAdapter & test with OAI
- fix lot of issues related to python 3.9 compatibility
- test_process.py is removed and going to reintroduce it on the next PR adding warmup features 
- fix tokenizers tests (TokenStream & BufferedTokenStream have STARTED/FINISHED events)

### CODE REVIEW

File: livekit-agents/livekit/agents/aio/channel.py [hunk 1]
```diff
@@ -150,9 +150,9 @@ def close(self) -> None:
     #        if self.empty():
     #            self._finished_ev.set()
 
-    @property
-    def closed(self) -> bool:
-        return self._closed
+    # @property
```
[the remark this answers is not in the document]
Comment (theomonnom) [reply livekit-agents/livekit/agents/aio/channel.py:153]: removed the function as it is a bad practice to use it (used nowhere anyway)

[the remark this answers is not in the document]
Comment (keepingitneil) [reply livekit-agents/livekit/agents/aio/channel.py:153]: should we just delete it?

[the remark this answers is not in the document]
Comment (theomonnom) [reply livekit-agents/livekit/agents/aio/channel.py:153]: yes

File: tests/test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+jiwer==3.0.4
```
Comment (keepingitneil) [inline tests/test-requirements.txt:1]: nice

**Review by keepingitneil — APPROVED** [review]
Nice PR, left some nits

File: examples/text-to-speech/elevenlabs_tts.py [hunk 3]
```diff
@@ -1,37 +1,105 @@
...
-    source = rtc.AudioSource(tts.sample_rate, tts.num_channels)
+    source = rtc.AudioSource(tts_11labs.sample_rate, tts_11labs.num_channels)
     track = rtc.LocalAudioTrack.create_audio_track("agent-mic", source)
     options = rtc.TrackPublishOptions()
     options.source = rtc.TrackSource.SOURCE_MICROPHONE
     await job.room.local_participant.publish_track(track, options)
 
     await asyncio.sleep(1)
     logging.info('Saying "Bonjour, comment allez-vous?"')
-    async for output in tts.synthesize("Bonjour, comment allez-vous?"):
```
Comment (davidzhao) [inline examples/text-to-speech/elevenlabs_tts.py:27]: we gonna do french by default? hah

**Review by davidzhao — COMMENTED** [review]
nice work! glad to have tests running again

File: examples/text-to-speech/elevenlabs_tts.py (hunk 3 above)
Comment (theomonnom) [reply examples/text-to-speech/elevenlabs_tts.py:27]: we have another example that isn't french, it is just to show that we can do whatever language here, but yeab I'll mb separate it in another file

**Comment by JARVISMindEngineer** [comment]
https://docs.livekit.io/agents/plugins/#VAD-and-StreamAdapter and README.md not update with this new feature.

`agents.tts.StreamAdapter(openai_tts, silero.VAD().stream(min_silence_duration=1.0))`, ** silero.VAD().stream(min_silence_duration=1.0) ** : Expect type Sentence Tokenizer, got VADStream instead;

**Pull request merged**